### PR TITLE
Meeting 127.md

### PR DESCRIPTION
**Pull request by Santhosh1692**

### CODE REVIEW

**Review by poojaranjan — COMMENTED** [review]
The meeting notes looks good to me, thanks!

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ The meetings are independent of any organization. However, Tim Beiko is a contra
...
 | --- | ------------------------------------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| 127 | November 26, 2021, 14:00 UTC          | [agenda](https://github.com/ethereum/pm/issues/416) | [notes](AllCoreDevs-Meetings/Meeting%20125.md) \| [Twitter](https://twitter.com/TimBeiko/status/1464262630139334658)                                             | [video](https://youtu.be/js4HLK4MyQI)                         |
 | 125 | October 29, 2021, 14:00 UTC          | [agenda](https://github.com/ethereum/pm/issues/401) | [notes](AllCoreDevs-Meetings/Meeting%20125.md) \| [Twitter](https://twitter.com/TimBeiko/status/1454188180609986564)                                             | [video](https://www.youtube.com/watch?v=5cOWjMAuReI)                         |
```
Comment (poojaranjan) [inline README.md:36]: Please add a row for meeting 126.

Comment (Santhosh1692) [reply README.md:37]: Done, updated

File: README.md [hunk 2]
```diff
@@ -32,6 +32,8 @@ The meetings are independent of any organization. However, Tim Beiko is a contra
...
 | --- | ------------------------------------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| 127 | November 26, 2021, 14:00 UTC          | [agenda](https://github.com/ethereum/pm/issues/416) | [notes](AllCoreDevs-Meetings/Meeting%20125.md) \| [Twitter](https://twitter.com/TimBeiko/status/1464262630139334658)                                             | [video](https://youtu.be/js4HLK4MyQI)                         |
+| 126 | November 12, 2021, 14:00 UTC          | [agenda](https://github.com/ethereum/pm/issues/407) | [notes](AllCoreDevs-Meetings/Meeting%20125.md) \| [Twitter](https://twitter.com/TimBeiko/status/1459240133752856577)                                             | [video](https://youtu.be/Kk-kZXazi18)                         |
```
Comment (poojaranjan) [inline README.md:36]:
```suggestion
| 126 | November 12, 2021, 14:00 UTC          | [agenda](https://github.com/ethereum/pm/issues/407) | [notes](AllCoreDevs-Meetings/Meeting%20126.md) \| [Twitter](https://twitter.com/TimBeiko/status/1459240133752856577)                                             | [video](https://youtu.be/Kk-kZXazi18)                         |
```

File: README.md [hunk 3]
```diff
@@ -32,6 +32,8 @@ The meetings are independent of any organization. However, Tim Beiko is a contra
 
 | №   | Date                                 | Agenda                                              | Notes                                                                                                                                                                    | Recording                                                                          |
 | --- | ------------------------------------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| 127 | November 26, 2021, 14:00 UTC          | [agenda](https://github.com/ethereum/pm/issues/416) | [notes](AllCoreDevs-Meetings/Meeting%20125.md) \| [Twitter](https://twitter.com/TimBeiko/status/1464262630139334658)                                             | [video](https://youtu.be/js4HLK4MyQI)                         |
```
Comment (poojaranjan) [inline README.md:35]:
```suggestion
| 127 | November 26, 2021, 14:00 UTC          | [agenda](https://github.com/ethereum/pm/issues/416) | [notes](AllCoreDevs-Meetings/Meeting%20127.md) \| [Twitter](https://twitter.com/TimBeiko/status/1464262630139334658)                                             | [video](https://youtu.be/js4HLK4MyQI)                         |
```

**Review by poojaranjan — APPROVED** [review]
Awesome, thank you!

**Pull request merged**